.git-hooks/pre-commit

```diff
@@ -37,6 +37,11 @@ my @kiwiModules = qw (
 	KIWIContentPlugin.pm
 	KIWIDescrPlugin.pm
 	KIWIEulaPlugin.pm
+    KIWIExt2Builder.pm
+    KIWIExt3Builder.pm
+    KIWIExt4Builder.pm
+    KIWIExtBuilderBase.pm
+    KIWIFilesystemBuilderBase.pm
 	kiwiFilesystemOptions.pm
```

Comment:
Could you fix the indentation for this file too ? all the rest looks fine now, thanks